src/test/java/seedu/address/ui/UiManagerTest.java

```diff
@@ -0,0 +1,224 @@
+package seedu.address.ui;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.GuiSettings;
+import seedu.address.logic.Logic;
+import seedu.address.logic.LogicManager;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
+import seedu.address.storage.JsonAddressBookStorage;
+import seedu.address.storage.JsonUserPrefsStorage;
+import seedu.address.storage.StorageManager;
+import seedu.address.testutil.Assert;
+
+public class UiManagerTest {
+
+    @TempDir
+    public Path temporaryFolder;
+
+    private UiManager uiManager;
+    private Logic logic;
+    private Model model = new ModelManager();
```

Comment:
just leaving a comment as a reference to our discussion jn: we can leave the instantiating of model to the `@BeforeEach` step to avoid the problem @Bluntsord mentioned (Needing to 'wipe' the model on each round).

Reply:
Think the problem wasnt wiping the model each round of test. It was wiping it within each test. So I think the clear command works

Reply:
Gotcha, I'll mark this convo as resolved.